lib/orderbook_sync.js

```diff
@@ -67,8 +69,21 @@ class OrderbookSync extends WebsocketClient {
       .catch(problems);
   }
 
+  // subscriptions changed -- possible new products
+  _newSubscription(data) {
+    const channel = data.channels.find(c => c.name === 'full');
+    channel && channel.product_ids
+      .filter(productID => !(productID in this.books))
+      .forEach(productID => this._newProduct(productID));
```

Comment:
Use same pattern as above (`this.productIDs.forEach(this._newProduct, this);`)?

Reply:
I dig it. Good eye. Done.